src/version.py

```diff
@@ -21,6 +20,6 @@
 # obtain version information
 from importlib.metadata import version
 
-__version__ = version("bg-cookiebutter")
```

Comment:
this has been automatically generated - this needs to be fixed - already have an issue